### DISCUSSION TITLE
plugin.ts中使用modifyRoutes后对应路由url中是本地路径，大佬们指导一下

### DISCUSSION THREAD

**Discussion by taiyangzaitianshang** [umijs/dumi#1994] [Q&A] [answered]
changelog.zh-CN.md在根目录，想通过约定式路由 docs/changelog-cn访问，在plugin下使用了如下API:

![image](https://github.com/umijs/dumi/assets/8625315/4c86192c-634f-4c07-bd17-370564ffbdaf)

但是部署后点击导航有如下错误：

![image](https://github.com/umijs/dumi/assets/8625315/a077699d-bc1c-4d8b-b443-4a08023e912b)

红框中的路径是本地绝对路径，dev的时候能正常导航（因为本地路径是对的）

请问大佬，是不是还需要其他配置呀

**Comment by PeachScript** [comment, marked as answer]
可能和 winPath 有关系，但建议用 embed 引用根目录的 changelog 而不是改路由表：https://d.umijs.org/guide/markdown#embed

**Reply by taiyangzaitianshang** [reply]
哦哦，好的，我试试，谢谢
